### ISSUE TITLE
 module 'keras_core.losses' has no attribute 'Reduction'

### ISSUE THREAD

**Issue by pranavvp16** [keras-team/keras-core#590]
seems there is no implementation for `reduction=keras.losses.Reduction.NONE` in losses

**Comment by fchollet** [comment]
Thanks for the report -- did you run into this while porting tf.keras code? What was the code?

**Comment by sampathweb** [comment]
`keras-core` doesn't use a enum of `losses.Reduction` and you could directly pass `reduction=None` for equivalent behavior - https://github.com/keras-team/keras-core/blob/main/keras_core/losses/loss.py#L75

**Comment by pranavvp16** [comment]
`loss_fn = keras.losses.SparseCategoricalCrossentropy(
            from_logits=True, reduction=keras.losses.Reduction.NONE
        )`
I tried with reduction=None, but it returns some different error.
I was trying to convert the NER using Transformers example code to keras_core code

**Comment by pranavvp16** [comment]
Sorry! there was some problem with my data, `"none"` and `None` works fine
